app/models/schedule.rb

```diff
@@ -0,0 +1,15 @@
+class Schedule < ApplicationRecord
+  validates :start_time, presence: true
+  validate :start_time, :date_cannot_be_before_start_time
+  validates :end_time, presence: true, comparison: { greater_than: :start_time }
+  validates :day, presence: true
```

Comment:
途中にあると見逃しそうなので一番下（上でもいい）に持ってきても良いかと思いました

```suggestion
  validates :end_time, presence: true, comparison: { greater_than: :start_time }
  validates :day, presence: true
  validate :start_time, :date_cannot_be_before_start_time
```

Reply:
どういう順番にした方が良いとかありますか？

Reply:
好みでいいかと思います！私は一番下に書きます！